### PR TITLE
feat(tests): add Playwright for e2e testing

### PR DESCRIPTION
**Pull request by olgapod**
## Description

-Setup Playwright testing framework for e2e tests

## Testing

* run `npm run playwright:install`
* run `npm run playwright:watch`

### CODE REVIEW

**Comment by chmanie** [comment]
Nice to see that you're adding playwright! I haven't used it but it seems quite promising!
I assume we won't be running these tests locally?

File: playwright.config.ts
Comment (iamsamgibbs) [inline playwright.config.ts:9]: I'm guessing this will be uncommented for something later on?

File: playwright.config.ts
Comment (iamsamgibbs) [inline playwright.config.ts:70]: Same here, guessing these will be uncommented when sorting out mobile tests?

**Review by iamsamgibbs — APPROVED** [review]
Congratulations on your first PR 🎉 

Nice job, this is all working as expected. Able to test in Playwright's UI:

Screenshot 2024-10-14 at 08 42 23

Tests are also passing in headless mode:

Screenshot 2024-10-14 at 08 46 36

Looking forward to this being expanded to include wallet connection etc and some proper tests! :)

**Review by rdig — CHANGES_REQUESTED** [review]
Please also remove all the `cypress` stuff in this PR

**Comment by rdig** [comment]
> Nice to see that you're adding playwright! I haven't used it but it seems quite promising! I assume we won't be running these tests locally and only on CI?

@chmanie I mean I also want us to retain the ability to run them locally, however, my main plan is to run them via nightlies, and with time, maybe on merged into `master`

_(I say with time, since I expect we're going to have a lot of tests which will take some time run)_

File: playwright.config.ts
Comment (olgapod) [reply playwright.config.ts]: I will remove them for now

**Comment by olgapod** [comment]
> Please also remove all the `cypress` stuff in this PR

done

**Review by rdig — APPROVED** [review]
All good here. Nice work!

And congratulations on your very first PR :tada: 

Screenshot from 2024-10-15 13-50-59

**Review by iamsamgibbs — APPROVED** [review]
Still all good!

Screenshot 2024-10-15 at 12 34 40

**Comment by rdig** [comment]
@olgapod please wait for 3 approvals before merging (even though the enforced rule is just two)

**Pull request merged**
